src/AzureFunctionPackage/Functions/Common/config.xml

```diff
@@ -87,5 +87,6 @@
   <!-- roslyn-analyzer branches -->
   <repo owner="dotnet" name="roslyn-analyzers">
     <merge from="2.9.x" to="master" />
```

Comment:
You can probably delete this entry: `<merge from="2.9.x" to="master" />`. We are not using 2.9.x branch anymore.